### PR TITLE
Improved TNEQ and TES processes

### PR DESCRIPTION
**Pull request by nagelt**
Changed some physical constants;
Added some new specific gas and solid thermal models;
Introduced temperature correction in reaction enthalpy

### CODE REVIEW

File: FEM/conversion_rate.cpp [hunk 1]
```diff
@@ -22,28 +22,34 @@ conversion_rate::conversion_rate(double T_solid,
...
+      tol_u (1.0 - tol_l),
+      tol_rho (0.1),
+      x(Eigen::VectorXd(1))
 {
 	update_param(T_solid, T_gas, p_gas, x_reactive, rho_s_initial, phi_S, delta_t, system);
 
 	if (system == FiniteElement::CaOH2)
 	{ // Definition auch in void CSolidProperties::SetSolidReactiveSystemProperties()
-		rho_low = 1656.0;
+        rho_low = 1665.1;
```
Comment (norihiro-w) [inline FEM/conversion_rate.cpp]: correct indentation

**Comment by norihiro-w** [comment]
:+1: besides minor comments about indentetations

File: FEM/eos.cpp [hunk 2]
```diff
@@ -563,6 +563,39 @@ double linear_heat_capacity(double T, int c)
...
+    //TODO: input file control: polynomial degree and coefficient array
+    //followed by a general loop here
+    double res(0.);
+    switch (c) {
+        case 1: //H2O
+            res = 1.5630770E+03;
+            res += 1.6037550E+00*T;
+            res += -2.9327840E-03*T*T;
+            res += 3.2161010E-06*T*T*T;
+            res += -1.1568270E-09*T*T*T*T;
```
Comment (TomFischer) [inline FEM/eos.cpp]: You could also use the Horner schema for the evaluation.

**Comment by wenqing** [comment]
:fast_forward:

**Comment by norihiro-w** [comment]
@wenqing is there any problem in your PR if we merge this?

**Comment by wenqing** [comment]
@norihiro-w There is not any shape function related stuff in this PR. Therefore you can merge it.

**Pull request merged**